webapp/views.py

```diff
@@ -506,6 +503,13 @@ def make_renewal(advantage, contract_info):
 
     renewal["renewable"] = False
 
+    if renewal["status"] == "done":
+        renewalModifiedDate = dateutil.parser.parse(renewal["lastModified"])
```

Comment:
Why using mixed case in here, change them to last_modified and one_hour_ago to be PEP8 compliant?